### ISSUE TITLE
Specifying 6.0.0 causes the build failing

### ISSUE THREAD

**Issue by ryonakano** [elementary/actions#59]
## What Happened

Specifying the version of granite as 6.0.0 in meson.build like [this](https://github.com/elementary/wingpanel-indicator-nightlight/commit/6a0b8be392ff5f246d080b5d237c78ce2ddd92a5#diff-30d8f6be6320feeacf686be94f48c70869b52630e01ea625f0f15adc0d57c3e4R35) causes [the build failing](https://github.com/elementary/wingpanel-indicator-nightlight/runs/1590615460#step:4:186) with the following error:

```
meson.build:25:0: ERROR: Invalid version of dependency, need 'granite' ['>=6.0.0'] found '5.5.0'.
```

Seems like the package name is libgranite**6** but its latest release for odin daily is **5.5.0**.

## Expected Behavior

The build succeeds (and the CI should pass).

**Comment by ryonakano** [comment]
Hmm, the gettext actions uses the latest odin daily but the latest available version of granite in odin seems to be 5.5.0:

```
ryo@ryo-pc:~/Projects$ apt-cache policy libgranite-dev 
libgranite-dev:
  Installed: 5.5.0+r1903+pkg117~daily~ubuntu6.1
  Candidate: 5.5.0+r1903+pkg117~daily~ubuntu6.1
  Version table:
 *** 5.5.0+r1903+pkg117~daily~ubuntu6.1 500
        500 http://ppa.launchpad.net/elementary-os/daily/ubuntu focal/main amd64 Packages
        100 /var/lib/dpkg/status
     5.3.0-1 500
        500 mirror://mirrors.ubuntu.com/mirrors.txt focal/universe amd64 Packages
```

Edit: Ah, I saw the actual project version is set to 6.0.0 just now :sweat_drops:

**Comment by ryonakano** [comment]
Seems like fixed by #60. Thank you @tintou! :tada:
